docs/Quickstart/connecting-to-the-outside.md

```diff
@@ -63,8 +63,8 @@ public class Nola extends Agent {
     }
 }
 ```
-
-We have created a simple helper function ```sendMessage``` to make it easier to send messages over Telegram. As each agent of class Nola communicates with exactly one user, we use that user’s Telegram ```chatId```.  Now that you’ve defined your first class, let’s write your first rule! We create a new file inside rules, with path ```rules/nola/Conversation.drl```:
+s
+We have created a simple helper function ```sendMessage``` to make it easier to send messages over Telegram. As each agent of class Nola communicates with exactly one user, we use that user’s Telegram ```chatId```.  Now that you’ve defined your first class, let’s write your first rule! We create a new folder inside rules and a new file inside the new folder, with path ```rules/nola/Conversation.drl```:
```

Comment:
maybe talk of directories instead of folders to keep consistent with remaining docs, and I would maybe put "and a new file inside it"